docs/1.getting-started/4.styling.md

```diff
@@ -416,7 +416,7 @@ SFC style blocks support preprocessors syntax. Vite come with built-in support f
 
 ::
 
-You can refer to the [Vite CSS docs](https://vitejs.dev/guide/features.html#css) and the [@vitejs/plugin-vue docs](https://github.com/vitejs/vite-plugin-vue/tree/main/packages/plugin-vue).
+You can refer to the [Vite CSS docs](https://vite.devuide/features.html#css) and the [@vitejs/plugin-vue docs](https://github.com/vitejs/vite-plugin-vue/tree/main/packages/plugin-vue).
```

Comment:
### Hi there,

I noticed that the link to the Vite documentation in the code has a typo. The current link is pointing to https://vite.devuide/features.html#css, but it should be updated to https://vite.dev/guide/features.html#css for accuracy.

Thank you for your attention to detail!

Best,